src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -218,34 +228,33 @@ export function loadDataStreams(data) {
 export function loadMappings(mappings) {
   perIndexTypes = {};
 
-  $.each(mappings, function (index, indexMapping) {
+  Object.entries(mappings).forEach(([index, indexMapping]) => {
     const normalizedIndexMappings = {};
 
     // Migrate 1.0.0 mappings. This format has changed, so we need to extract the underlying mapping.
-    if (indexMapping.mappings && _.keys(indexMapping).length === 1) {
+    if (indexMapping.mappings && Object.keys(indexMapping).length === 1) {
       indexMapping = indexMapping.mappings;
     }
 
-    $.each(indexMapping, function (typeName, typeMapping) {
+    Object.entries(indexMapping).forEach(([typeName, typeMapping]) => {
       if (typeName === 'properties') {
         const fieldList = getFieldNamesFromProperties(typeMapping);
         normalizedIndexMappings[typeName] = fieldList;
       } else {
         normalizedIndexMappings[typeName] = [];
       }
     });
-
     perIndexTypes[index] = normalizedIndexMappings;
   });
 }
 
 export function loadAliases(aliases) {
```

Comment:
related to my previous comment. Maybe it make sense to create a separate files for each entries an use one interface for loading / initialization of each entry?

My motivation here to refactor that code to TS and make it a little bit more readable 

@cjcenizal @sabarasaba

Reply:
Created a separate issue for converting the file to ts https://github.com/elastic/kibana/issues/129716